### DISCUSSION TITLE
Could Winget integrate with manifest creating support?

### DISCUSSION THREAD

**Discussion by taozuhong** [microsoft/winget-cli#3539] [General]
Could Winget integrate with manifest creating support?

**Comment by mdanish-kh** [comment]
The team has a planned feature to integrate winget-create into the WinGet CLI

- #989

If you need help with manifest creation, currently there are a couple [manifest authoring tools](https://github.com/microsoft/winget-pkgs#authoring-a-manifest) available to help you get started.
